docroot/themes/custom/yukonca_glider/src/sass/layouts/_home.layout.scss

```diff
@@ -3,3 +3,528 @@
     @apply bg-white;
   }
 }
+
+.aurora-town {
+  position: relative;
+  z-index: -1;
+
+  margin-top: -6px;
+}
+
+.aurora--main {
+  position: relative;
+
+  display: inline-block;
+  overflow: hidden;
+
+  width: 100%;
+  height: 50px;
+
+  transition: all 0.4s ease;
+
+  @media (min-width: 767px) {
+    position: absolute;
+    top: 0;
+    right: 0;
+
+    width: 100%;
+    height: 200px;
+
+    transition: all 0.4s ease;
+  }
+
+  &:before {
+    position: absolute;
+    top: 0;
+    right: -100px;
+
+    display: block;
+    overflow: hidden;
+
+    width: 300px;
+    height: 50px;
+
+    content: "";
+    transition: all 0.4s ease;
+
+    background-image: url("../../images/svg/Aurora-small-mobile.svg");
+    background-repeat: no-repeat;
+
+    @include media-breakpoint-up(md) {
+      position: absolute;
+      top: -30px;
+      right: -60%;
+
+      width: 100%;
+      height: 185px;
+
+      transition: all 0.4s ease;
+
+      background-image: url("../../images/svg/Aurora-main.svg");
+      background-size: contain;
+    }
+  }
+}
+
+.aurora--mini {
+  display: inline-block;
+  overflow: hidden;
+
+  width: 100%;
+  padding: 50px 0;
+
+  &:before {
+    position: relative;
+
+    display: block;
+
+    width: 260px;
+    height: 45px;
+    margin: 0 auto;
+
+    content: "";
+
+    background-image: url("../images/Aurora-mini.svg");
```

Comment:
seems wrong path to the image. All images are in svg folder, so should be something like this "../../images/svg/Aurora-mini.svg"

Reply:
Yes that's right.  I fixed it now.